apps/web/src/components/GalleryEditor/GalleryEditorContext.tsx

```diff
@@ -496,66 +496,62 @@ export function GalleryEditorProvider({
     track,
   ]);
 
-  const publishGallery = useCallback(
-    async (_caption: string | null) => {
-      const galleryId = query.galleryById.dbid;
-      const caption = _caption?.trim() ? _caption : null;
+  const publishGallery = useCallback(async () => {
+    const galleryId = query.galleryById.dbid;
 
-      if (!galleryId) {
-        reportError('Tried to publish a gallery without a gallery id');
-        return;
-      }
+    if (!galleryId) {
+      reportError('Tried to publish a gallery without a gallery id');
+      return;
+    }
 
-      const payload = {
-        galleryId,
-        caption,
-        editId: editSessionID,
-      };
+    const payload = {
+      galleryId,
+      caption: null,
+      editId: editSessionID,
+    };
+
+    track('Publish Gallery', payload);
 
-      track('Publish Gallery', payload);
+    try {
+      const { publishGallery } = await publish({
+        variables: {
+          input: payload,
+        },
+      });
 
-      try {
-        const { publishGallery } = await publish({
-          variables: {
-            input: payload,
-          },
-        });
+      if (publishGallery?.__typename !== 'PublishGalleryPayload' || !publishGallery.gallery) {
```

Comment:
the diff here is kind of weird, this is just moved up a few lines because I removed a few lines above